# Raw acceleration values
float64 acc_x_raw
float64 acc_y_raw
float64 acc_z_raw

# Gripper Position
float64 gripper_joint_position
float64 gripper_joint_velocity
float64 gripper_joint_effort
